### PR TITLE
Irminger 6 cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New cal csvs for instruments and Glider sensors that are on Irminger 6.  **Please note that Collin's Pull Request #48 also contains a few cal csvs for Glider sensors on Irminger 6 cruise.  We need to get that pull request reviewed and approved/merged soon.  I will work on reviewing it tomorrow, but it will need 2nd reviewer as well.

### CODE REVIEW

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20181220.csv [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
....92, 274.74, 275.55, 276.37, 277.18, 278, 278.81, 279.63, 280.44, 281.26, 282.07, 282.88, 283.7, 284.51, 285.33, 286.14, 286.96, 287.77, 288.59, 289.4, 290.22, 291.03, 291.85, 292.66, 293.48, 294.29, 295.11, 295.92, 296.74, 297.55, 298.37, 299.18, 299.99, 300.81, 301.62, 302.44, 303.25, 304.07, 304.88, 305.69, 306.51, 307.32, 308.14, 308.95, 309.76, 310.58, 311.39, 312.21, 313.02, 313.83, 314.65, 315.46, 316.27, 317.09, 317.9, 318.71, 319.53, 320.34, 321.15, 321.96, 322.78, 323.59, 324.4, 325.21, 326.03, 326.84, 327.65, 328.46, 329.27, 330.09, 330.9, 331.71, 332.52, 333.33, 334.14, 334.95, 335.77, 336.58, 337.39, 338.2, 339.01, 339.82, 340.63, 341.44, 342.25, 343.06, 343.87, 344.68, 345.49, 346.3, 347.11, 347.91, 348.72, 349.53, 350.34, 351.15, 351.96, 352.76, 353.57, 354.38, 355.19, 355.99, 356.8, 357.61, 358.42, 359.22, 360.03, 360.84, 361.64, 362.45, 363.25, 364.06, 364.86, 365.67, 366.47, 367.28, 368.08, 368.89, 369.69, 370.5, 371.3, 372.1, 372.91, 373.71, 374.51, 375.32, 376.12, 376.92, 377.72, 378.52, 379.33, 380.13, 380.93, 381.73, 382.53, 383.33, 384.13, 384.93, 385.73, 386.53, 387.33, 388.13, 388.93, 389.73, 390.53, 391.32, 392.12, 392.92, 393.72, 394.51, 395.31, 396.11]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01065__20181220.csv]: Agrees with Python calibration parser. Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20190215.csv [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20190215.csv]: Agrees with Python calibration parser. Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20190313.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
...61, 272.42, 273.23, 274.05, 274.86, 275.67, 276.49, 277.3, 278.11, 278.93, 279.74, 280.55, 281.37, 282.18, 282.99, 283.81, 284.62, 285.43, 286.24, 287.06, 287.87, 288.68, 289.5, 290.31, 291.12, 291.93, 292.75, 293.56, 294.37, 295.18, 295.99, 296.81, 297.62, 298.43, 299.24, 300.05, 300.87, 301.68, 302.49, 303.3, 304.11, 304.92, 305.74, 306.55, 307.36, 308.17, 308.98, 309.79, 310.6, 311.41, 312.22, 313.03, 313.84, 314.65, 315.46, 316.27, 317.08, 317.89, 318.7, 319.51, 320.32, 321.13, 321.94, 322.75, 323.56, 324.37, 325.18, 325.98, 326.79, 327.6, 328.41, 329.22, 330.02, 330.83, 331.64, 332.45, 333.25, 334.06, 334.87, 335.68, 336.48, 337.29, 338.09, 338.9, 339.71, 340.51, 341.32, 342.12, 342.93, 343.73, 344.54, 345.34, 346.15, 346.95, 347.76, 348.56, 349.37, 350.17, 350.97, 351.78, 352.58, 353.38, 354.18, 354.99, 355.79, 356.59, 357.39, 358.2, 359, 359.8, 360.6, 361.4, 362.2, 363, 363.8, 364.6, 365.4, 366.2, 367, 367.8, 368.6, 369.4, 370.19, 370.99, 371.79, 372.59, 373.39, 374.18, 374.98, 375.78, 376.57, 377.37, 378.16, 378.96, 379.76, 380.55, 381.35, 382.14, 382.93, 383.73, 384.52, 385.32, 386.11, 386.9, 387.69, 388.49, 389.28, 390.07, 390.86, 391.65, 392.45, 393.24, 394.03, 394.82]",
```
Comment (reedan88) [inline calibration/NUTNRB/CGINS-NUTNRB-01107__20190313.csv]: Agrees with Python parser. Looks good.

File: calibration/CTDGVM/CGINS-CTDGVM-09231__20170531.csv [hunk 4]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+9231,CC_calibration_date,1496188800000,sensor calibration date (milliseconds since 1970-01-01)
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/CTDGVM/CGINS-CTDGVM-09231__20170531.csv]: Correct.

File: calibration/PARADM/CGINS-PARADM-50185__20170713.csv [hunk 5]
```diff
@@ -0,0 +1,2 @@
+serial,name,value,notes
+50185,CC_bsipar_par_scaling,1,
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/PARADM/CGINS-PARADM-50185__20170713.csv]: Correct.

File: calibration/PCO2WC/CGINS-PCO2WC-C0122__20190603.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0122,CC_cala,0.0653,source_file PCO2W-C_SAMI-CO2_SN_C0122_Calibration_Files_2019-06-03.zip > C122_05_CalCert.pdf
+C0122,CC_calb,0.5242,
+C0122,CC_calc,-1.4387,
+C0122,CC_calt,14.0548,
+C0122,CC_ea434,19706,constant
+C0122,CC_ea620,34,constant
+C0122,CC_eb434,3073,constant
+C0122,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/PCO2WC/CGINS-PCO2WC-C0122__20190603.csv]: Correct.

File: calibration/PCO2WC/CGINS-PCO2WC-C0121__20190603.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0121,CC_cala,0.0848,source_file PCO2W-C_SAMI-CO2_SN_C0121_Calibration_Files_2019-06-03.zip > C121_05_CalCert.pdf
+C0121,CC_calb,0.3976,
+C0121,CC_calc,-1.2390,
+C0121,CC_calt,14.1459,
+C0121,CC_ea434,19706,constant
+C0121,CC_ea620,34,constant
+C0121,CC_eb434,3073,constant
+C0121,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/PCO2WC/CGINS-PCO2WC-C0121__20190603.csv]: Correct.

File: calibration/PCO2WC/CGINS-PCO2WC-C0105__20190603.csv [hunk 8]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+C0105,CC_cala,0.0462,source_file PCO2W-C_SAMI-CO2_SN_C0105_Calibration_Files_2019-06-03.zip > C105_05_CalCert.pdf
+C0105,CC_calb,0.6256,
+C0105,CC_calc,-1.5758,
+C0105,CC_calt,14.0850,
+C0105,CC_ea434,19706,constant
+C0105,CC_ea620,34,constant
+C0105,CC_eb434,3073,constant
+C0105,CC_eb620,44327,constant
```
[the remark this answers is not in the document]
Comment (snwhite) [reply calibration/PCO2WC/CGINS-PCO2WC-C0105__20190603.csv]: Correct.

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20190313.csv (hunk 3 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01107__20190313.csv]: Looks good, but not on Irminger 6.

File: calibration/NUTNRB/CGINS-NUTNRB-01065__20181220.csv (hunk 1 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01065__20181220.csv]: Looks good.

File: calibration/NUTNRB/CGINS-NUTNRB-01107__20190215.csv (hunk 2 above)
Comment (snwhite) [reply calibration/NUTNRB/CGINS-NUTNRB-01107__20190215.csv]: Looks good, but not on Irminger 6.

**Pull request merged**